### ISSUE TITLE
cfme specific strings need to be removed from repo or made into configuration

### ISSUE THREAD

**Issue by Fryguy** [ManageIQ/bugzilla_mirror#1]
- cfme-\* flags
- settings.yml (odd name, btw, I would expect queries.yml) has flag specifics.  Should that file even be committed?

**Comment by abellotti** [comment]
Yes, this makes sense. moving all _cfme_ out of repos. Configuration driven to tailor our build systems/apps for cfme. One thing though, to easily rebuild any app, we could save these config in a seperate private repo, maybe cfme_build_configs, one dir per app. There's probably other options.

**Comment by kbrock** [comment]
The rest of the world tends to be using env variables...
But into a yaml file sounds good too.
